CHANGELOG.md

```diff
@@ -1,53 +1,7 @@
-Desktop: update SAC fields and other statistics when editing cylinders
-Desktop: Reconnect the variations checkbox in planner
-Desktop: add support for dive mode on CSV import and export
-Desktop: fix profile display of planned dives with surface segments
```

Comment:
The top four deleted items are bug fixes, which are not in current release.

Reply:
oops. thanks for catching that.